master/internal/routes.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+type redirect struct {
+	src    string
+	dest   string
+	method string
+	code   int
+}
+
+var redirects = [...]redirect{
+	{
+		src:    "/",
+		dest:   "/det",
+		method: http.MethodGet,
+		code:   http.StatusMovedPermanently,
+	},
+}
+
+func setupEchoRoutes(m *Master) {
```

Comment:
maybe "setupEchoRedirects"?

Reply:
I'm imagining we move more of the route definitions out of core.go and here hence the name. wdy think

Reply:
I was sort of thinking of moving them too, but not all to the same function (since I dont think that's strictly possible for all of them). I was going to suggest going the route of new api and having `api_x` files for all. Not sure.

Reply:
I don't know enough about Go but would anything that's inside master.Run should be transferable here, it's the same package and both have access to `*Master`. ? 

with that said I agree probably separate files would be even better.

Reply:
yeh it'll be transferable here. I'm fine with this, i just think until that happens the name is confusing/misleading.

Reply:
done